src/crimson/os/seastore/cache.h

```diff
@@ -1689,13 +1689,13 @@ class Cache {
 	if (likely(extent->state == CachedExtent::extent_state_t::CLEAN_PENDING)) {
 	  extent->state = CachedExtent::extent_state_t::CLEAN;
 	  /* TODO: crc should be checked against LBA manager */
-	  extent->last_committed_crc = extent->get_crc32c();
+	  extent->last_committed_crc = extent->calc_crc32c();
```

Comment:
Suggest to move it to the common logic below, remove the addressed TODO comments, and replace with another comment, e.g.:

```c++
// crc will be checked against LBA leaf entry for logical extents,
// or check against in-extent crc for physical extents.
```